apps/frontend/src/pages/admin/billing/[id].vue

```diff
@@ -337,6 +388,34 @@ async function refundCharge() {
   refunding.value = false;
 }
 
+async function modifyCharge() {
+  modifying.value = true;
+  try {
+    await useBaseFetch(`billing/subscription/${selectedCharge.value.id}`, {
+      method: "PATCH",
+      body: JSON.stringify({
+        cancelled: cancel.value,
+      }),
+      internal: true,
+    });
+    data.$notify({
+      group: "main",
+      title: "Resubscription request submitted",
+      text: "If the server is currently suspended, it may take up to 10 minutes for another charge attempt to be made.",
+      type: "success",
+    });
+    await refreshCharges();
+  } catch (err) {
+    data.$notify({
```

Comment:
Use explicit addNotification composable (import it), $notify will be removed in the future.

Reply:
Group is also not a thing anymore